app/models/user.rb

```diff
@@ -0,0 +1,31 @@
+require 'bcrypt'
+
+class User
```

Comment:
What the hell has happened to your indentation in this file?

Is `User` not related to `Peep`?